src/lib/components/pile/pile.svelte

```diff
@@ -1,3 +1,10 @@
+<script lang="ts" context="module">
+  export type PileComponent = {
+    component: typeof SvelteComponent;
+    props: Record<string, unknown>;
+  };
```

Comment:
Parents can infer the type of `Pile`'s props using `ComponentProps`, no need to export them explicitly.